hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestSecureOzoneManager.java

```diff
@@ -85,8 +83,6 @@ public void init() throws Exception {
     omId = UUID.randomUUID().toString();
     conf.setBoolean(OZONE_ACL_ENABLED, true);
     conf.setBoolean(OZONE_SECURITY_ENABLED_KEY, true);
-    conf.setTimeDuration(OZONE_OM_OPEN_KEY_EXPIRE_THRESHOLD,
-        2, TimeUnit.SECONDS);
```

Comment:
Does the OpenKey Clean Update Service cause this CI to fail?

Reply:
I don't think so. IIRC, this test is flaky before.

Reply:
There is no problem in TestSecureOzoneManager with or without HDDS-4123

https://github.com/kaijchen/ozone/actions/runs/2454403397
https://github.com/kaijchen/ozone/actions/runs/2454495538